code/modules/mob/living/living.dm

```diff
@@ -703,6 +703,29 @@ Thanks.
 							pulling.Move(T, get_dir(pulling, T))
 							if(M && secondarypull)
 								M.start_pulling(secondarypull)
+						//this is the gay blood on floor shit -- Added back -- Skie
+							if (M.lying && (prob(M.getBruteLoss() / 6)))
```

Comment:
The probability needs increased drastically.  It is extremely difficult to get to proc.